api/REST-API.md

```diff
@@ -50,7 +50,8 @@ Stateless  client/server  protocol:  each  HTTP  contains  all  the  necessary
 - Cacheable – Cache  constraints  require  that  the  data within  a  response  to  a  request  be  implicitly  or  explicitly  labeled  as cacheable  or  non-cacheable.  If  a  response  is  cacheable,  then  a  client  cache  is  given  the  right  to  reuse  that  response  data  for later,  equivalent  requests.
 - Uniform interface – By  applying  the  software  engineering  principle  of  generality  to  the  component  interface,  the  overall  system architecture  is  simplified  and  the  visibility  of  interactions  is  improved.  In  order  to  obtain  a  uniform  interface,  multiple architectural  constraints  are  needed  to  guide  the  behavior  of  components.  REST  is  defined  by  four  interface  constraints: identification  of  resources;  manipulation  of  resources  through  representations;  self-descriptive  messages;  and,  hypermedia  as  the engine  of  application  state.
 - Layered  system – The  layered  system  style  allows  an  architecture  to  be  composed  of  hierarchical  layers  by  constraining  component behavior  such  that  each  component  cannot  “see”  beyond  the  immediate  layer  with  which  they  are  interacting.
-- Code  on  demand – REST  allows  client  functionality  to  be  extended  by  downloading  and  executing  code  in  the  form  of applets  or  scripts.  This  simplifies  clients  by  reducing  the  number  of  features  required  to  be  pre-implemented.
+- Code  on  demand – REST  allows  client  functionality  to  be  extended  by  downloading  and  executing  code  in  the  form  of applets  or  scripts.  This  simplifies  clients  by  reducing  the  number  of  features  required  to  be  pre-implemented.[(restfulapi.net)](https://restfulapi.net/)
```

Comment:
This report said there were more sources... is this really all from that same site?  https://smallseotools.com/view-report/55935ef38455fd043acbc748820c891e/

Reply:
@alodahl yes I am sure that this part is also from the same source. it's the last point in the list in the given link. I have removed the word optional from the description and hence it must be showing some other sites too. But I took this data from the site I have mentioned.